src/components/DataView/DataViewColumDrawer/DataViewColumnDrawer.styled.ts

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+import theme from "@root/theme";
+
+export const ColumnTitle = styled.h2`
+  color: ${theme.newColors.almostBlack["100"]};
+  font-size: 20px;
+  font-weight: ${theme.fontWeight.normal};
+  line-height: 24px;
+  margin: 0 0 24px 0;
+`;
+
+export const StyledWrapper = styled.div`
+  display: flex;
+  margin-top: 30px;
+
+  & > .left {
+    border-right: 2px solid ${theme.newColors.grey2["100"]};
+    flex: 1;
+    padding: 0 32px 0 40px;
```

Comment:
Left uses padding and right uses margin, I recommend keeping all of them with margin, if we do that the drawer becomes 693px just like the figma